api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -163,47 +174,81 @@ public long getQueryBatchSize() {
     return jdbcTemplate.query(
         String.format(
             "SELECT \n"
-                + "  u.about_you,\n"
-                + "  u.area_of_research,\n"
-                + "  u.compliance_training_bypass_time,\n"
-                + "  u.compliance_training_completion_time,\n"
-                + "  u.compliance_training_expiration_time,\n"
-                + "  u.contact_email,\n"
-                + "  u.creation_time,\n"
-                + "  u.current_position,\n"
-                + "  u.data_access_level,\n"
-                + "  u.data_use_agreement_bypass_time,\n"
-                + "  u.data_use_agreement_completion_time,\n"
-                + "  u.data_use_agreement_signed_version,\n"
-                + "  u.demographic_survey_completion_time,\n"
-                + "  u.disabled,\n"
-                + "  u.era_commons_bypass_time,\n"
-                + "  u.era_commons_completion_time,\n"
-                + "  u.family_name,\n"
-                + "  u.first_registration_completion_time,\n"
-                + "  u.first_sign_in_time,\n"
-                + "  u.free_tier_credits_limit_days_override,\n"
-                + "  u.free_tier_credits_limit_dollars_override,\n"
-                + "  u.given_name,\n"
-                + "  u.last_modified_time,\n"
-                + "  u.professional_url,\n"
-                + "  u.two_factor_auth_bypass_time,\n"
-                + "  u.two_factor_auth_completion_time,\n"
                 + "  u.user_id,\n"
-                + "  u.email AS username,\n"
-                + "  a.city,\n"
-                + "  a.country,\n"
-                + "  a.state,\n"
-                + "  a.street_address_1,\n"
-                + "  a.street_address_2,\n"
-                + "  a.zip_code,\n"
-                + "  via.institution_id AS institution_id,\n"
-                + "  via.institutional_role_enum,\n"
-                + "  via.institutional_role_other_text\n"
+                + "  GROUP_CONCAT(DISTINCT u.about_you) AS about_you,\n"
```

Comment:
why is it necessary to use GROUP_CONCAT?

Reply:
There are some repeated field that we want to aggregate into the user reporting table. And to use GROUP_CONCAT, all field has to added that key words. Some background:
https://pmi-engteam.slack.com/archives/C0150CWF6FP/p1613604172255000?thread_ts=1612902414.216000&cid=C0150CWF6FP

Reply:
makes sense, thanks

Reply:
Can you please add a comment to this effect in the method here?